cypress/integration/pages/testsForAllPages.js

```diff
@@ -117,6 +122,11 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
                   ? 'article'
                   : 'website';
 
+                // For Ukrainian pages with Russian content we want to use the `ru_UA` locale
+                const contentLocale =
+                  appConfig[config[service].name][variant].metaLocale ||
+                  appConfig[config[service].name][variant].locale;
+
```

Comment:
You can revert the changes to this test now that `metaLocale` is not needed. You should be able to use `appConfig[config[service].name][variant].locale` below as it was previously.